### DISCUSSION TITLE
Change Text

### DISCUSSION THREAD

**Discussion by dhe365** [squidfunk/mkdocs-material#3419] [Q&A] [answered]
Hi, love the theme! Been using mkdocs for a while but I wanna switch over to this theme. Question: How do I change the text marked on screenshot marked within the red rectangle? It's on the main page. Tried searching for it in vscode but couldn't find where to edit it?!

![alt](https://i.imgur.com/jZpjhme.png)

Thank you!

**Comment by squidfunk** [comment, marked as answer]
https://github.com/squidfunk/mkdocs-material/blob/12a016bcf9658cb782af6fffed1a17c0f474cc78/src/overrides/home.html#L80-L81

**Reply by dhe365** [reply]
Thank you for the quick reply! :) Hmm, I tried modifying it, but the website/text stays the same - even after saving and serving the docs again with `mkdocs serve`. Am I missing something?

Thank you!

Edit: Got it, I git cloned the repo and had to make the changes to /material/ instead of /src/.

**Reply by squidfunk** [reply]
You need to go through the [build process](https://squidfunk.github.io/mkdocs-material/customization/#theme-development) when directly making changes to the theme. Note that it's not advisable to make changes to the `material` folder, as they will be overwritten on the next merge.
